crates/telio-traversal/src/connectivity_check/cross_ping_check.rs

```diff
@@ -286,7 +322,7 @@ impl CrossPingCheckTrait for CrossPingCheck {
 }
 
 impl<E: Backoff> State<E> {
-    fn get_connectivty_check_state<'a>(
+    fn get_connectivity<'a>(
```

Comment:
To me the old name looks clearer

Reply:
whoops, actually my bad, I was just fixing a typo `..tivty` -> `tivity` but somehow removed the remaining part of the name :D Fixed that